app/javascript/stylesheets/application.css

```diff
@@ -0,0 +1,120 @@
+@import "tailwindcss/base";
+@import "tailwindcss/components";
+@import "tailwindcss/utilities";
+
+@layer base {
+  /* Font Sizing and weight */
+  body {
+    @apply text-lg md:text-xl xl:text-2xl;
+  }
+  
+  h1 {
+    @apply text-6xl md:text-7xl 2xl:text-8xl font-bold !important;
+  }
+  h2 {
+    @apply text-4xl md:text-5xl 2xl:text-6xl font-medium !important;
+  }
+  h3, h4 {
+    @apply text-3xl md:text-4xl 2xl:text-5xl font-bold !important;
+  }
+  h5, h6 {
+    @apply text-2xl md:text-3xl 2xl:text-4xl !important;
+  }
+
```

Comment:
Titles were very large in normal computer view.

![imagen](https://user-images.githubusercontent.com/12243331/104645022-27de3080-5674-11eb-8f4c-c675e7478a5c.png)

![imagen](https://user-images.githubusercontent.com/12243331/104645079-36c4e300-5674-11eb-8bc3-0cb33c4ae542.png)

I don't know how to assign styles by sheets?

That is, one for home # index, another for market_places?

Reply:
That is because `h1`s are not meant to be put there, for better SEO and Markup there should only be one `h1` per page and/or section; most of the features detailed in the MVP are pop-ups, modals and pages that are inserted and combined into `application.html.erb` without any specialized sectioning such as `<section>`or `<article>`, so it will be a little bit tricky to use multiple `h1` without sectioning the HTML and naming those headings with something representative and useful for SEO (something beyond the name of the selected marketplace). In the Figma file every style for text is linked with the name of the recommended tag to use.

Reply:
Fine. We will have to make an issue to refactor the frontend and adapt it to the styles assigned in Figma.